src/components/social.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import Social from './social.js';
+import reactGA from 'react-ga';
+
+var ShareProgressButton = React.createClass({
+  componentDidMount: function() {
+    var shareProgressHolder = document.querySelector(".share-progress-holder");
+    var shareProgressButton = shareProgressHolder.querySelector("." + this.props.shareProgress);
+    if (shareProgressButton) {
+      shareProgressHolder.removeChild(shareProgressButton);
+      this.shareProgressContainer.appendChild(shareProgressButton);
```

Comment:
where do we bind this.shareProgressContainer?

Reply:
It bound on line 25 currently:

```
<div ref={(container) => { this.shareProgressContainer = container; }}></div>
```